### ISSUE TITLE
cpmfyui 上可以克隆语音吗？

### ISSUE THREAD

**Issue by Song367** [AIFSH/CosyVoice-ComfyUI#9]

**Comment by av8dtop** [comment]
Error occurred when executing CosyVoiceNode:

[ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Non-zero status code returned while running Add node. Name:'/Add_2' Status Message: D:\a\_work\1\s\onnxruntime\core/providers/cpu/math/element_wise_ops.h:560 onnxruntime::BroadcastIterator::Append axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 1500 by 6382


File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\custom_nodes\CosyVoice-ComfyUI\nodes.py", line 131, in generate
output = self.cosyvoice.inference_zero_shot(tts_text, prompt_text, prompt_speech_16k)
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\custom_nodes\CosyVoice-ComfyUI\cosyvoice\cli\cosyvoice.py", line 59, in inference_zero_shot
model_input = self.frontend.frontend_zero_shot(i, prompt_text, prompt_speech_16k)
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\custom_nodes\CosyVoice-ComfyUI\cosyvoice\cli\frontend.py", line 228, in frontend_zero_shot
speech_token, speech_token_len = self._extract_speech_token(prompt_speech_16k)
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\custom_nodes\CosyVoice-ComfyUI\cosyvoice\cli\frontend.py", line 158, in _extract_speech_token
speech_token = self.speech_tokenizer_session.run(None, {self.speech_tokenizer_session.get_inputs()[0].name: feat.detach().cpu().numpy(),
File "C:\Users\av8dt\Documents\ComfyUI-aki-v1.2\python\lib\site-packages\onnxruntime\capi\onnxruntime_inference_collection.py", line 220, in run
return self._sess.run(output_names, input_feed, run_options)

**Comment by bluevisor** [comment]
用自带的3s极速复刻workflow就行，我跑通了。

**Comment by dcp-dev** [comment]
Hey did you manage to fix the issue? I got the same even if using the `workflows/3sclone_workflow.json` as suggested in the previous answar
